examples/react/start-basic-apollo-client/app/components/ApolloProvider.tsx

```diff
@@ -0,0 +1,98 @@
+import {
+  WrapApolloProvider,
+  QueryEvent,
+  DataTransportContext,
+} from '@apollo/client-react-streaming'
+import { useRouter } from '@tanstack/react-router'
+import {
+  ComponentProps,
+  useCallback,
+  useEffect,
+  useId,
+  useMemo,
+  useRef,
+} from 'react'
+
+const APOLLO_EVENT_PREFIX = '@@apollo.event/'
+const APOLLO_HOOK_PREFIX = '@@apollo.hook/'
+
+const handledEvents = new WeakSet<object>()
+
+export const ApolloProvider = WrapApolloProvider((props) => {
```

Comment:
This file is essentially all that's needed for Apollo Client streaming SSR integration with fetch-as-you-render (`useSuspenseQuery` etc).

Reply:
This would end up in it's own package, I guess?